### ISSUE TITLE
Different Results Generated from 0.10.1 MXNET R Package and 0.9.4 MXNET R Package

### ISSUE THREAD

**Issue by yuzhouchen92** [apache/incubator-mxnet#7525]
Dear Users and Developers,
 
I worked in a project about data mining which using convolutional neural networks based on MXNET R package. And when I use the old version of MXNET R package 0.9.4, I got a pretty good result. However, the error rate generated by updated version( 0.10.1) was two times higher than the error rate generated by old version (0.9.4). I had checked the source code in github (https://github.com/apache/incubator-mxnet/tree/master/R-package), and found that developers added more mx functions, more parameters in some functions. So I guess that maybe these cause the differences between two results generated by old version package and updated version package. Also, I kept the same random seeds on the top of my codes. 

My question is that whether there are some errors/mistakes in old MXNET R package? 

Thanks!
Best

**Comment by jeremiedb** [comment]
I haven't noticed issues between 0.9.4 and 0.10 and I'm not aware of changes in R package that were tied to bug, though corrections in the backend library might explain the difference. 
Just a guess here, but if the model has BatchNorm, there have been some adjustments to its parameters. If using SGD optimizer, could be worth trying training with a different learning rate or use another optimizer like adadelta to compare the versions behavior.

**Comment by thirdwing** [comment]
@yuzhouchen92 Can you share an example?

**Comment by yuzhouchen92** [comment]
@jeremiedb  Yes, my model concludes BatchNorm, and using SGD as my optimizer. I had tried different learning rate, momentum, and used other optimizers, but the result generated from 0.10.1 R package was still worse than the old one. Sure, I will re-check the BatchNorm layer. Thanks for your suggestion!

**Comment by yuzhouchen92** [comment]
@thirdwing Because my data was provided by one bank, so I maybe could not put the whole thing here. I will share one toy example here later.

**Comment by ankkhedia** [comment]
@nswamy @sandeep-krishnamurthy Could you please label the issue as Pending Requester Info

**Comment by ankkhedia** [comment]
@sandeep-krishnamurthy @nswamy Could you please close this issue due to inactivity

@yuzhouchen92 Please feel free to reopen if closed in error
